doc/specs/#1595 - Suggestions UI/Snippets.md

```diff
@@ -255,26 +287,30 @@ refresh that list as the user changes directories.
   the actions living in an entirely separate map of CWD->actions, the
   keybindings in the main map won't be able to easily get to them. See also
   [Security considerations](#security) for more.
-* If the Snippets pane or Suggestions UI is opened with `local` suggestions as a
-  source, then we'll just append the appropriate list of suggestions for the
-  active control's CWD.
+* If the Snippets pane or Suggestions UI is opened with `snippets` as a source,
+  then we'll just append the appropriate list of suggestions for the active
+  control's CWD.
   * We don't need to have the control raise an event when the CWD changes - we
     can lazy-load these actions when a UI element that requires it is first
     invoked.
-* The Command Palette is trickier, since it binds directly to the action map.
-  We'd need to be able to freely modify that map at runtime, which might be
-  prohibitively annoying. <!-- * If we want these actions to show up in the
-  Command Palette, we'll need to:
-  * We'll stash these actions in the action map as they're loaded. (in
-    `Terminal.Settings.Model`)
-  * We'll need to be able to dynamically remove them at runtime from the map (in
-    `Terminal.Settings.Model`) -->
+* These snippets will not be included in the Command Palette.

```

Comment:
So, I know we reached this conclusion in the review yesterday. Can we include why here and if that's a permanent solution or just one that's deprioritized?

For reference, I _do_ want them to be in the command palette (or at least have a mode in the command palette dedicated to snippets). But I still don't fully understand why we're entirely dropping it. So trying to be respectful and open-minded here.